src/SourceBuild/tarball/content/Directory.Build.props

```diff
@@ -181,6 +181,7 @@
     <PortableRid Condition="'$(TargetOS)' == 'FreeBSD'">freebsd-$(Platform)</PortableRid>
     <PortableRid Condition="'$(TargetOS)' == 'OSX'">osx-$(Platform)</PortableRid>
     <PortableRid Condition="'$(TargetOS)' == 'Linux'">linux-$(Platform)</PortableRid>
+    <PortableRid Condition="$(TargetRid.StartsWith('linux-musl')) or $(TargetRid.StartsWith('alpine')) or $(TargetRid.StartsWith('void'))">linux-musl-$(Platform)</PortableRid>
```

Comment:
Can you explain the void check? How does that relate to musl?

Reply:
AFAIK, Void Linux (https://voidlinux.org/) distro can also use musl (or glibc, seems to be user-selectable).

Reply:
Right, indeed Void can be both musl and glibc. I've removed this part, as indeed we can't assume that void is musl.